tests/test-extensions/petstore-extension/src/main/java/org/mule/test/petstore/extension/WatermarkingPetAdoptionSource.java

```diff
@@ -33,7 +33,7 @@
 
   @Parameter
   @org.mule.runtime.extension.api.annotation.param.Optional(defaultValue = "false")
-  protected boolean watermark;
```

Comment:
Was there any bug that prevents to use this value here, but allows it on master? Is this already identified?